tck/src/main/java/ee/jakarta/tck/concurrent/spec/signature/SignatureTestServlet.java

```diff
@@ -25,13 +31,45 @@
 @WebServlet("SignatureTestServlet")
 public class SignatureTestServlet extends TestServlet {
 	private static final long serialVersionUID = 1L;
+	
+	private static final TestLogger log = TestLogger.get(SignatureTestServlet.class);
 
 	public void testSignatures(HttpServletRequest req, HttpServletResponse resp) throws Throwable {
 		
 		//Ensure that jimage directory is set.
 		//This is where modules will be converted back to .class files for use in signature testing	
-		assertNotNull(System.getProperty("jimage.dir"), "The system property jimage.dir must be set in order to run this test.");
+		assertNotNull(System.getProperty("jimage.dir"), "The system property jimage.dir must be set in order to run the Signature test.");
+		
+		//Ensure user to running on JDK 11, this has to be exact, since the sigtest-maven-plugin output will change depending on JDK level
+		assertTrue(Integer.parseInt(System.getProperty("java.specification.version")) == 11, "The signature tests must be run on an application server using JDK 11 exactly.");
```

Comment:
Will it be necessary to also be able to run on Java 17 in order to report TCK results on that level as well?

Reply:
That is a good question.  It would be easy enough to generate a separate signature file for Java 17.

Reply:
Since we are passing in the -IgnoreJDKClass option with each JDK class to ignore, we should be able to just use the `Java 11` signature map file for running on `Java 11` and `Java 17`.